src/v/kafka/server/group_metadata_migration.cc

```diff
@@ -687,11 +687,11 @@ ss::future<> group_metadata_migration::start(ss::abort_source& as) {
           mlog.info,
           "kafka_internal/group topic does not exists, activating "
           "consumer_offsets feature");
-        try {
-            co_await activate_feature(as);
-        } catch (const ss::abort_requested_exception&) {
-            // ignore abort requested exception, we are shutting down
-        }
+
+        ssx::spawn_with_gate(_background_gate, [this, &as]() -> ss::future<> {
+            return activate_feature(as).handle_exception(
+              [](std::exception_ptr) {});
```

Comment:
I know that we should avoid this (catch std::exception type)

But I can not get type of this exception. 
When I print it I got `std::exception`. Looks like we can not avoid this `handle_exception`.